### PR TITLE
Issue marker cleanup

### PR DESCRIPTION
**Pull request by ianbjacobs**
-- Added issue markers for FPWD.
-- Cleaned up some old issue markers

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -319,6 +319,9 @@
           readonly attribute PaymentManager paymentManager;
         };
       </pre>
+
+      	<p class="issue" title="Issue 127 - Do we need Payment Manager and Payment Handler?"><a href="https://github.com/w3c/webpayments-payment-apps-api/issues/127">Issue 127</a>: The distinction is unclear. A proposal is to remove the phrase Payment Manager from the spec, renaming the "PaymentManager" interface the "PaymentHandler" interface.</p>
+
```
Comment (adrianhopebailie) [inline index.html:324]: Suggest the issue can be closed. Can remove when done.

File: index.html [hunk 2]
```diff
@@ -1310,7 +1329,7 @@
         }));
       });
       </pre>
-      	  <p class="issue"><a href="https://github.com/w3c/webpayments-payment-apps-api/issues/128">Issue 128</a>: The Web Payments Working Group plans to revisit these two examples.</p>
+      	  <p class="issue" title="Issue 128 - Revisit examples"><a href="https://github.com/w3c/webpayments-payment-apps-api/issues/128">Issue 128</a>: The Web Payments Working Group plans to revisit these two examples.</p>
```
Comment (adrianhopebailie) [inline index.html:1332]: We should provide examples before FPWD or take the existing examples out. Most developers will jump directly to the examples and not see the issue marker.

**Review by adrianhopebailie — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Can we PLEASE stop rushing in merges without adequate review. 

I would again kindly ask that we put a process in place and stop adding things on a whim to the spec. As someone who has to implement this, it's super frustrating if I don't get to review stuff. It also wastes your time, because there are faster ways of doing things. Please see: 

https://github.com/w3c/respec/wiki/Referencing-GitHub-issues-in-your-spec

**Comment by adrianhopebailie** [comment]
@marcoscaceres there is a process but I'm hearing you say you don't like the process.

In the interests of getting things done and being pragmatic about the current maturity of this spec we are merging following review by any of the editors.

What do you suggest we change about that process?

**Comment by ianbjacobs** [comment]
Hi all,

I think we should review the editorial process, especially, @marcoscaceres, if you become an active editor.

My understanding of what we are doing to get to FPWD is mostly cleanup and putting issue markers in the spec. I would not characterize what we are doing as adding things on a whim.

Once we get to FPWD, and as implementations increase, we may well want to adjust our process. I think there is strong agreement that there should be review before substantive additions. Let's figure out (after FPWD) how to do that effectively. I also would like to make a pitch for the ability to fix obvious editorial bugs without review, at least until the spec is much more mature.

My goal right now is to stabilize the spec in its current form to get to FPWD, as we agreed at the FTF meeting. I have no other edits planned, but imagine there may be some small changes based on next Tuesday's discussion.

Ian

**Comment by marcoscaceres** [comment]
> On 16 Apr 2017, at 12:47 am, ianbjacobs <notifications@github.com> wrote:
> 
> Hi all,
> 
> I think we should review the editorial process, especially, @marcoscaceres, if you become an active editor.
> 

I promise I will after we get Payment Request to CR. As much as I want to, I don't have the bandwidth to help edit this one right now :( 

> My understanding of what we are doing to get to FPWD is mostly cleanup and putting issue markers in the spec. I would not characterize what we are doing as adding things on a whim.
> 

I get edgy if things move too fast. In the spec, we always wait until at least one another implementer has a chance to review and approve the changes. Preferably, we should have two reviewers who are experts in whatever is being added/modified doing the review. 

> Once we get to FPWD, and as implementations increase, we may well want to adjust our process. I think there is strong agreement that there should be review before substantive additions. Let's figure out (after FPWD) how to do that effectively. I also would like to make a pitch for the ability to fix obvious editorial bugs without review,
> 
They have to be super obvious (e.g., broken markup). All other changes, no matter how trivial you believe they might be, must be reviewed: there are a lot of moving parts in these specs, so small changes tend to have ripple effects. 
> at least until the spec is much more mature.
> 
We need to flip the process, IMO. Immature things should never go in the spec: only in branches, which after a lot of review and iterations go into the spec. That's how we've been doing for the PR spec. 

Take a look at the way Domenic and I have been working in the PR spec - in addition to our own back and forth, we always make sure changes are also approved by at least 1 person from Microsoft. 

We should identify key stakeholders who can review changes. 

> My goal right now is to stabilize the spec in its current form to get to FPWD, as we agreed at the FTF meeting. I have no other edits planned, but imagine there may be some small changes based on next Tuesday's discussion.
> 
Sounds fine - but let's switch to using the same process as the PR spec ASAP.

**Comment by ianbjacobs** [comment]
@marcoscaceres wrote:

   > In the spec, we always wait until at least one another implementer has a chance to review and 
   > approve the changes. Preferably, we should have two reviewers who are experts in whatever is    
   > being added/modified doing the review.

I'm ok with that moving forward but that is not what we had previously agreed to to get to FPWD.

  > Immature things should never go in the spec: only in branches, which after a lot of review and 
  > iterations go into the spec. That's how we've been doing for the PR spec.

I'm fine to do that once we get to FPWD. 

 > Let's switch to using the same process as the PR spec

Is there documentation?

Ian

**Comment by marcoscaceres** [comment]
> Is there documentation?
> 
Yes, what I wrote above. It's pretty easy: don't merge things without thorough review - always get review from multiple implementers. If we can't get review from implementers (or implementer interest/consensus ) - immediately stop. 

If pull request includes code, make sure that code has been run (even in just a prollyfill). 

That's it.

**Pull request merged**